CHANGELOG.rst

```diff
@@ -4,6 +4,20 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+2.3.0
+--------
+
+**Added**
+
+* ``ProjectIdentifier``, ``ProjectCode``, ``ProjectSpace`` and ``Project`` to describe QBiC projects
+* ``ProjectApplication`` to describe a project application for registration at QBiC's data management platform
```

Comment:
Please also provide the complete identifier since this can lead to problems later (we have multiple Person classes for example).
```suggestion
* ``life.qbic.datamodel.dtos.projectmanagement.ProjectIdentifier``, ``life.qbic.datamodel.dtos.projectmanagement.ProjectCode``, ``life.qbic.datamodel.dtos.projectmanagement.ProjectSpace`` and ``life.qbic.datamodel.dtos.projectmanagementProject`` to describe QBiC projects
* ``life.qbic.datamodel.dtos.projectmanagementProjectApplication`` to describe a project application for registration at QBiC's data management platform
```

```suggestion
* ``ProjectIdentifier``, ``ProjectCode``, ``ProjectSpace`` and ``Project`` to describe QBiC projects in ``life.qbic.datamodel.dtos.projectmanagement``
* ``ProjectApplication`` to describe a project application for registration at QBiC's data management platform in ``life.qbic.datamodel.dtos.projectmanagement``
```